contents/docs/experiments/sample-size-running-time.mdx

```diff
@@ -0,0 +1,40 @@
+export const SampleSizeDeterminationDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/user-guides/experimentation/sample-size-determination-dark.png"
+
+When creating an experiment, we provide recommended running times and sample sizes based on the parameters you choose. Please note that values serve as estimates for how long to run the experiment. You can end the experiment early if you observe a significant effect.
+
+For trend experiments, we use [Lehr's equation](http://www.columbia.edu/~cjd11/charles_dimaggio/DIRE/styled-4/code-12/#poisson-distributed-or-count-data) to determine sample sizes.
+
+<FormulaScreenshot
+  imageLight={LehrEquationLight}
+  imageDark={LehrEquationDark} 
+  alt="Lehr equation" 
+  classes="rounded"
+/>
+
+Here, lambda1 (λ1) represents the baseline count data from the past two weeks, and lambda2 (λ2) is calculated as `baseline count + MDE * (baseline count)`. The MDE (Minimum Detectable Effect) is the minimum acceptable improvement you select in the UI.
+
+For funnel experiments, we use the general [Sample size determination](https://en.wikipedia.org/wiki/Sample_size_determination) formula, with 80% power and 5% significance level. The formula is as follows:
+
+<FormulaScreenshot
+  imageLight={SampleSizeDeterminationLight}
+  imageDark={SampleSizeDeterminationDark} 
+  alt="Sample size determination" 
+  classes="rounded"
+/>
+
+These values serve as estimates for how long to run the experiment. It's possible to conclude experiments early if you observe a significant effect.
+
+## Minimum Detectable Effect (MDE)
+When setting up an experiment, the Minimum Detectable Effect (MDE) is a required parameter for estimating sample size and running time. The MDE represents the smallest change or difference that you want to be able to detect in your experiment. Essentially, it's the minimum improvement you would consider significant.
+
+To make things easier, we set a default MDE value to get you started, we provide an estimated MDE value by default. This allows you to create experiments quickly without getting stuck on this step. However, we encourage you to review and adjust this value based on your specific goals. If you have a particular MDE in mind, feel free to set it accordingly.
+
+It's important to understand that the recommended sample size decreases as the MDE increases. In other words, if you are looking for a smaller improvement, you'll need a larger sample size to detect it. This is because smaller effects are harder to identify and require more data to ensure the results are statistically significant. Conversely, if you're looking for a larger improvement, you won't need as many samples because the effect is easier to spot. This relationship helps ensure that your experiment is sensitive enough to detect the changes you're interested in.
```

Comment:
```suggestion
It's important to understand that the recommended sample size decreases as the MDE increases. In other words, if you are looking for a smaller improvement, you'll need a larger sample size to detect it. This is because smaller effects are harder to identify and require more data to ensure the results are statistically significant. 

Conversely, if you're looking for a larger improvement, you won't need as many samples because the effect is easier to spot. This relationship helps ensure that your experiment is sensitive enough to detect the changes you're interested in.
```